sdk/servicebus/Azure.Messaging.ServiceBus/src/Client/ServiceBusClient.cs

```diff
@@ -132,10 +132,10 @@ protected ServiceBusClient()
         /// </remarks>
         public ServiceBusClient(string connectionString, ServiceBusClientOptions options)
```

Comment:
nit: This should check for null.

Reply:
We do:
`_options = options?.Clone() ?? new ServiceBusClientOptions();`